ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -19,6 +19,7 @@ da_scala_library(
     ],
     deps = [
         "//daml-lf/data",
+        "//language-support/scala/bindings",
```

Comment:
I'm a bit surprised the server has to depend on the client bindings, how come?

Reply:
`TimeProvider` lives there. Is there a better place?

Reply:
If it's supposed to be shared between clients and servers, probably `ledger-api-common`.

Reply:
I'll move it there and remove the dep.

Reply:
This is more complicated than it looks; I suggest we do it elsewhere.